pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/zookeeper-operator/index.md

```diff
@@ -14,6 +14,8 @@ excerpt: Information about the ZooKeeper Operator
 
 Follow these steps to install the ZooKeeper operator in a workspace. This procedure results in a ZooKeeper operator running in a workspace namespace, ready to manage and create ZooKeeper clusters in any project namespaces. See the [ZooKeeper custom resource documentation](../../../../../projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/zookeeper/) for more information on creating ZooKeeper clusters.
 
+<p class="message--note"><strong>NOTE: </strong>You should only install the ZooKeeper operator once per workspace.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Only install the ZooKeeper operator once per workspace.</p>
```

Reply:
fixed in 7195cc0c82201695a22e0befdc2b61ff9f2661c8